api/interfaces/common/VsoBaseInterfaces.ts

```diff
@@ -52,15 +54,34 @@ export interface IBasicCredentials {
     password: string;
 }
 
+export interface IHttpClient {
```

Comment:
I think we need to export both this and IRequestInfo because we're using them as parameters in other interfaces that are being exported. If I don't export them, an error is thrown ~Parameter 'httpClient' of method from exported interface has or is using private name 'IHttpClient'.~